Suggest a PR for traits rather than issue (#1065)
pi-base/data: .github/ISSUE_TEMPLATE/new-trait-suggestion.md

> <!-- Modify the below template for your trait suggestion: -->
> 
> <!-- NOTE! If your proposal is not likely to need much discussion, you should
> consider opening a Pull Request to implement this suggestion directly (assuming
> you know how). -->
> 

Author: ```suggestion
<!-- NOTE! If your proposal is not likely to need much discussion, you can
consider opening a Pull Request to implement this suggestion directly, if you feel
comfortable doing so. -->

<!-- Modify the below template for your trait suggestion: -->

```
